drafts/latest/tables/deprecated-properties-for-data-service.html

```diff
@@ -91,7 +97,9 @@
 
 <aside class="note">
 <p>Until GeoDCAT-AP version 1.0.1 [[GeoDCAT-AP-20160802]], property <code>rdfs:comment</code> was used with class <a href="#service"><code>dctype:Service</code></a> (replaced by class <code>dcat:DataService</code> since this version of GeoDCAT-AP) to specify the spatial resolution of a Data Service, since a more specific property was not available in reference vocabularies.</p>
-<p>[[VOCAB-DCAT-2]] has defined a specific property to specify spatial resolution as distance in metres, namely, <code>dcat:spatialResolutionInMeters</code>. Therefore <code>dcat:spatialResolutionInMeters</code> replaces <code>rdfs:comment</code> starting from this version of GeoDCAT-AP, but only when spatial resolution is expressed as distance in metres.</p>
+<p>[[VOCAB-DCAT-2]] and [[?SDW-BP]] recommend the use of [[VOCAB-DQV]] for the specification of different types of spatial resolution, via property <code>dcat:hasQualityMeasurement</code>. Moreover, [[VOCAB-DCAT-2]] has defined a specific property to specify spatial resolution as distance in metres, namely, <code>dcat:spatialResolutionInMeters</code>.</p>
```

Comment:
[[VOCAB-DCAT-2]]  refers to DQV for expressing quality statements, but AFAIR, it does not explicitly refer to dqv:hasQualityMeasurement for spatial resolution.
Besides correcting  "dcat:hasQualityMeasurement" with "dqv:hasQualityMeasurement", I would consider rephrasing the note a little ...

```suggestion
<p>[[?SDW-BP]] recommends the use of [[VOCAB-DQV]] for the specification of different types of spatial resolution, via property <code>dqv:hasQualityMeasurement</code>.  [[VOCAB-DCAT-2]] reuses  [[VOCAB-DQV]]  for expressing quality statements and it  has defined a specific property to specify spatial resolution as distance in metres, namely, <code>dcat:spatialResolutionInMeters</code>.</p>
```

Reply:
The referred statement is the one at the end of [§6.6.4 Property: spatial resolution](https://www.w3.org/TR/vocab-dcat-2/#Property:dataset_spatial_resolution) of DCAT2 - quoting:

> The range of this property is a decimal number representing a length in meters. This is intended to provide a summary indication of the spatial resolution of the data as a single number. More complex descriptions of various aspects of spatial precision, accuracy, resolution and other statistics can be provided using the Data Quality Vocabulary [VOCAB-DQV]. 

There is indeed no explicit reference to `dqv:hasQualityMeasurement`, but the [DQV section about prevision and accuracy](https://www.w3.org/TR/vocab-dqv/#ExpressDatasetAccuracyPrecision) is using that property in the examples showing how to specify spatial resolution. Said that, the current wording is indeed ambiguous, so maybe we can replace "via property `dqv:hasQualityMeasurement`" with "where property `dqv:hasQualityMeasurement` is used".

The resulting revision:

> <p>[[VOCAB-DCAT-2]] and [[?SDW-BP]] recommend the use of [[VOCAB-DQV]] for the specification of different types of spatial resolution, where property <code>dcat:hasQualityMeasurement</code> is used. Moreover, [[VOCAB-DCAT-2]] has defined a specific property to specify spatial resolution as distance in metres, namely, <code>dcat:spatialResolutionInMeters</code>.</p>

 @riccardoAlbertoni , would this make sense to you?

Reply:
Yes, it works for me. Thanks.